corehq/blobs/migrate_metadata.py

```diff
@@ -178,8 +178,19 @@ def __init__(self, obj, key, domain, reindexer):
         self.blobs = {"": BlobMetaRef(key=key, **reindexer.blob_kwargs(obj))}
         self.external_blobs = self.blobs
 
+    def __repr__(self):
+        return "<%s %s domain=%s id=%s>" % (
+            type(self).__name__,
+            self.doc_type,
+            self.domain,
+            self._id,
+        )
+
     @property
     def _id(self):
+        # NOTE unlike couch documents, this is different from `doc["_id"]`,
+        # the value used to set `BlobMeta.parent_id`. This value should
+        # only be used to identify the record in in case of error.
```

Comment:
Added this comment to explain the confusing difference between `doc["_id"]` and `obj._id`. Also added `__repr__` above for better console error output.